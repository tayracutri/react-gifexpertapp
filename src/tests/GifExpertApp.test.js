import { shallow } from "enzyme";
import { GifExpertApp } from "../GifExpertApp";
import React from 'react';

describe('test on gifexpertapp', () => {
    test('should match the snapshot', () => {
        const wrapper = shallow(<GifExpertApp />);

        expect(wrapper).toMatchSnapshot();
    });

    test('should show the categories', () => {
        const categories = ['Dani', 'Tay'];
        const wrapper = shallow(<GifExpertApp defaultCategories={categories} />);

        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('GifGrid').length).toBe(categories.length);
    })

})
