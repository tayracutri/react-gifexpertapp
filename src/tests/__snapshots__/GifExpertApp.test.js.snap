// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on gifexpertapp should match the snapshot 1`] = `ShallowWrapper {}`;

exports[`test on gifexpertapp should show the categories 1`] = `ShallowWrapper {}`;
